CHANGELOG.md

```diff
@@ -26,6 +35,11 @@ _October 25, 2023_
     - Enabled by `Components.default.messageAutoTranslationEnabled`
 ### 🐞 Fixed
 - Do not mark a channel as read if the app is in background [#2832](https://github.com/GetStream/stream-chat-swift/pull/2832)
```

Comment:
Hm, are you sure you correctly merged the changelog? This is from the previous release.

Reply:
Fixed ✅